comparative_benchmark/xla_hlo/run_benchmarks.py

```diff
@@ -192,10 +196,12 @@ def _run_compiler_benchmark_cpu(
   compile_time_latency = float(matches[0]) if matches else None
 
   matches = CPU_LATENCY_REGEXP.findall(result_text)
-  assert len(matches) == benchmark_iterations, (
-      f"Expected to find {benchmark_iterations} latencies but found "
-      f"{len(matches)} instead:\n{result_text}")
-  latencies = [float(match) * 1000 for match in matches]
+  if len(matches) == benchmark_iterations:
+    latencies = [float(match) * 1000 for match in matches]
+  else:
+    error_string = f"Expected to find {benchmark_iterations} latencies but found {len(matches)} instead:\n{result_text}"
+    print(error_string)
```

Comment:
nit: We can check and only print if `verbose == True`

Reply:
Added the check